src/Utils/GetCategories.js

```diff
@@ -1,5 +1,6 @@
 export const uniqueCategory = (data, category) => {
-  const categoryName = data.map((eachProduct) => eachProduct[category]);
+
+  const categoryName = data?.map((eachProduct) => eachProduct[category]);
```

Comment:
I think this was supposed to be videos instead of product

Reply:
Done